src/cli.js

```diff
@@ -311,19 +314,19 @@ async function runCmd (argv, stdout, stderr) {
       break;
 
     default:
-      errInvalidCommand(args._[0]);
+      errInvalidCommand(args._[0], 2);
   }
 
   function errTooManyArguments (cmd) {
-    nccError(`Error: Too many ${cmd} arguments provided\n${usage}`);
+    nccError(`Error: Too many ${cmd} arguments provided\n${usage}`, 2);
   }
 
   function errFlagNotCompatible (flag, cmd) {
-    nccError(`Error: ${flag} flag is not compatible with ncc ${cmd}\n${usage}`);
+    nccError(`Error: ${flag} flag is not compatible with ncc ${cmd}\n${usage}`, 2);
```

Comment:
I've extended the use of the `2` exit code here for CLI validation errors, as was provided for the help output already.

I'm not sure who implemented this code in the first place, but it seems a useful separation and helps to work with the code-based tests, but I can certainly remove it too and move this to an integration test rather as well.